peps/pep-0768.rst

```diff
@@ -368,6 +392,33 @@ proposed interface.
 By maintaining compatibility with existing security frameworks, this design
 ensures that adopting the new interface requires no changes to established.
 
+Security scenarios
+------------------
+
+* For an external attacker, the ability to write to arbitrary memory in a
+  process is already a severe security issue. This interface does not introduce
+  any new attack surface, as the attacker would already have the ability to
+  execute arbitrary code in the process. This interface behaves in exactly
+  the same way as existing debuggers, and does not introduce any new additional
+  security risks.
+* For an attacker who has gained arbitrary memory write access to a process but
+  not arbitrary code execution, this interface does not provide a way to escalate
+  to arbitrary code execution as long as the attacker has not gained control over
+  additional machine resources external to the program. 
+  
+Additionally, the fact that the code to be executed is gated by the interpreter's
+audit hooks means that the execution of the code can be monitored and controlled
+by system administrators. This means that even if the attacker has compromised the
+application **and the filesystem**, leveraging this interface for malicious
+purposes provides a very risky proposition for an attacker, as they risk
+exposing their actions to system administrators that could not only detect the
+attack but also take action to prevent it. Finally, is important to note that
```

Comment:
```suggestion
attack but also take action to prevent it.

Finally, is important to note that
```

Plus rewrapping... not doing all that in a PR comment, sorry 😆